src/containers/PracticeAnswer/PracticeAnswer.js

```diff
@@ -0,0 +1,240 @@
+            alignItems: "flex-end",
+          }}
+        >
+          <Body style={{ lineHeight: "1.2em" }}>
+            ยินดีด้วย! คุณได้รับ 1 เหรียญ
+          </Body>
+          <Coin />
+        </div>
+      );
+    } else {
+      if (!correct) {
+        return (
+          <div
+            style={{
+              marginTop: "104px",
+              marginBottom: "16px",
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "flex-end",
+            }}
+          >
+            <Body style={{ lineHeight: "1.2em", visibility: "hidden" }}></Body>
+          </div>
+        );
+      }
+    }
+  };
+
+  const arrowHolder = () => {
+    if (firstClick) {
+      if (solution.length === staticSolution.length) {
+        return (
+          <ShiftDiv>
+            <div style={{ marginTop: "20px" }}>
+              <Button
+                type="custom"
+                border="none"
+                color={COLOR.WHITE}
+                backgroundColor={
+                  correct ? `${COLOR.CELERY}` : `${COLOR.MANDARIN}`
+                }
+              >
+                ทำต่อ
+              </Button>
+            </div>
+          </ShiftDiv>
+        );
+      } else {
+        if (correct) {
+          return (
+            <ShiftDiv>
+              <ShiftLeft
+                src={Correct_Backward}
+                onClick={() => backward(solution)}
+              />
+              <ShiftRight
+                src={Correct_Forward}
+                onClick={() => forward(solution, staticSolution)}
+              />
+            </ShiftDiv>
+          );
+        } else {
+          return (
+            <ShiftDiv>
+              <ShiftLeft
+                src={Incorrect_Backward}
+                onClick={() => backward(solution)}
+              />
+              <ShiftRight
+                src={Incorrect_Forward}
+                onClick={() => forward(solution, staticSolution)}
+              />
+            </ShiftDiv>
+          );
+        }
+      }
+    } else {
+      return <ShiftDiv></ShiftDiv>;
+    }
+  };
+
+  const handleFirstClick = () => {
+    setFirstClick(true);
+  };
+
+  useEffect(() => {
+    // TODO: Get the solution and store each line in staticSolution
+    //   axios.get().then((res) => {
+    //       setStaticSolution(res)
+    //     })
+  });
+
+  return (
+    <Container answer={correct} onClick={handleFirstClick}>
+      <div>
+        <CenterDiv>
+          {/* {correct ? <Sign src={Correct} /> : <Sign src={Incorrect} />} */}
+          <Sign answer={correct} />
+        </CenterDiv>
+        <CenterDiv>
+          <Title answer={correct}>{title}</Title>
+        </CenterDiv>
+        {firstClick ? (
+          <SolutionDiv>
+            <ul style={{ listStyle: "none", padding: 0 }}>
+              {solution.map((line) => {
+                // TODO: Replace Math.random() with solution.id after it has one
+                return (
```

Comment:
ไม่น่าต้องมี return ข้างใน map ป่ะนะ

Reply:
ทำไมถึงจะไม่มีนะ map มัน return ค่าไม่ใช่รึ

Reply:
เพราะมันอยู่ใน return อยู่แล้วง่ะ + เราก็ลอก map จากลิ้งนี้แล้วมันไม่ได้ใส่นะะ (https://reactjs.org/docs/lists-and-keys.html)
<img width="759" alt="Screen Shot 2563-12-07 at 18 46 02" src="https://user-images.githubusercontent.com/42530415/101347312-7867ae80-38bc-11eb-9eee-8d8437d5cc9b.png">

Reply:
ก็เราใส่ปีกกาอะ มันก็เลยต้องใส่ return มันเหมือนกันหนิ